src/overlays/actors/ovl_kaleido_scope/z_kaleido_collect.c

```diff
@@ -176,7 +174,7 @@ void KaleidoScope_DrawQuestStatus(GlobalContext* globalCtx, GraphicsContext* gfx
             KaleidoScope_SetCursorVtx(pauseCtx, sp216 * 4, pauseCtx->questVtx);
 
             if ((pauseCtx->state == 6) && (pauseCtx->unk_1E4 == 0) && (pauseCtx->cursorSpecialPos == 0)) {
-                if ((sp216 >= 6) && (sp216 < 0x12)) {
+                if (sp216 >= 6 && sp216 < 0x12) {
```

Comment:
I'm guessing that it's preferred to keep the brackets even on something this small.

Reply:
its decopmilers discretion at this point, no set rule. I would personally keep them but we havent been enforcing anything thus far